library/ix-dev/community/sftpgo/item.yaml

```diff
@@ -0,0 +1,6 @@
+icon_url: https://github.com/drakkan/sftpgo/blob/main/img/logo.png?raw=true
+categories:
+  - storage
+screenshots: []
```

Comment:
Btw we are not interested in adding screenshots anymore?

Reply:
I didn't find any generic screenshots, all I could find was just shots from users for bug reports.